databricks/koalas/tests/test_series_plot.py

```diff
@@ -67,8 +67,8 @@ def test_bar_plot(self):
         pdf = self.pdf1
         kdf = self.kdf1
 
-        ax1 = pdf['a'].plot("bar", colormap='Paired')
-        ax2 = kdf['a'].plot("bar", colormap='Paired')
+        ax1 = pdf['a'].plot(kind='bar', colormap='Paired')
+        ax2 = kdf['a'].plot(kind='bar', colormap='Paired')
```

Comment:
i think you could leave it as is... the failure should be caused by that missing function, if you remove it, i think CI will pass